index.html

```diff
@@ -85,6 +85,268 @@ <h1>
+        </div>
+
+        <div class="about_section">
+          <section class="about_me">
+            <div class="about_info">
+              <div class="about_me_title">
+                <h3>About<br>
+                  Myself</h3>
+              </div>
+              <div class="about_desc">
+                <p>
+                  Hello I’m a software developer! I can help<br>
+                   you build a product , feature or website <br>
+                   Look through some of my work and<br>
+                    experience! If you like what you see and<br>
+                     have a project you need coded, don’t<br>
+                      hestiate to contact me.
+                </p>
+              </div>
+              <div class="let_connect">
+                <a href="#">
+                  <h4>LET’S CONNECT</h4>
+                </a>
+              </div>
+                <div class="social-images">
+                  <ul>
+                    <li>
+                      <a href="#">
+                        <img src="images/github.png" alt="github-icon" />
+                      </a>
+                    </li>
+                    <li>
+                      <a href="#">
+                        <img src="images/Linkedinicon.svg" alt="Linkedin-icon" />
+                      </a>
+                    </li>
+                    <li>
+                      <a href="#">
+                        <img src="images/twitter.png" alt="twitter-icon" />
+                      </a>
+                    </li>
+                    <li>
+                      <a href="#">
+                        <img src="images/Vector(1).svg" alt="line-icon" />
+                      </a>
+                    </li>
+                    <li>
+                      <a href="#">
+                        <img src="images/Vector(2).svg" alt="socail-icon" />
+                      </a>
+                    </li>
+                  </ul>
+              </div>
+              <div class="btn_resume">
+                <a href="#">Get My Resume</a>
+              </div>
+            </div>
+
+            <div class="skills">
+
+                <div class="language">
+                    <h3>Languages</h3>
+                </div>
+      
+                <div class="languages">
+                  <ul>
+                    <li>
+                      <a href= "#">
+                        <img src="images/JS1.png" alt="" /> <p>JavaScript</p>
+                      </a>
+                          
+                    </li>
+
+                    <li>
+                      <a href="#">
+                        <img src="images/HTML1.png" alt="" /> <p>HTML</p>
+                      </a>
+                    </li>
+                 
+                    <li>
+                      <a href="#">
+                        <img src="images/CSS1.png" alt="" /> <p>JavaScript</p>
+                      </a>
+                    </li>
+                  </ul>
```

Comment:
- [ ] [Optional] Kindly follow the Figma design and use the drop-down arrows for language, framework, skills and make changes to it
### your design
![hiss](https://user-images.githubusercontent.com/81727988/170819885-4cb62fd7-29b7-41ae-852b-eb5b25405cca.png)
### Figma design
![figma1111](https://user-images.githubusercontent.com/81727988/170819893-b5c156bb-cf01-4a02-b9b1-b2e885a46d39.png)